indico/types/submission.py

```diff
@@ -0,0 +1,24 @@
+from indico.types import BaseType
+
+
+class Submission(BaseType):
+    """
+    A Submission in the Indico Platform
+
+    Attributes:
+        id (int): the Submission id
+        dataset_id (int): the Dataset id
+        workflow_id (int): the Workflow id
+        status (str): status of the submission
```

Comment:
Going off of the pretty extensive docstring here: https://github.com/IndicoDataSolutions/indico-client-python/blob/master/indico/types/jobs.py
I think a lot more information can be added here?